File: androiddevkit/src/main/java/com/ams/androiddevkit/utils/services/logging/LoggingService.java
package com.ams.androiddevkit.utils.services.logging;

public interface LoggingService {
    void e(String tag, String msg);
    void i(String tag, String msg);
    void d(String tag, String msg);
    void w(String tag, String msg);
    void e(String tag, String msg, Throwable ex);
    void v(String tag, String msg);
    void stackTrack(Throwable throwable);
    void stackTrack(Throwable throwable, boolean force);
    void logUser(String userId, String userName);
    void logUser(long userId, String userName);
    void longInfo(String tag, String str);
}

File: androiddevkit/src/main/java/com/ams/androiddevkit/baseClasses/services/LoggingService.kt
package com.ams.androiddevkit.baseClasses.services

interface LoggingService {
    abstract fun e(tag: String, msg: String)
    abstract fun i(tag: String, msg: String)
    abstract fun d(tag: String, msg: String)
    abstract fun w(tag: String, msg: String)
    abstract fun e(tag: String, msg: String, ex: Throwable)
    abstract fun v(tag: String, msg: String)
    abstract fun stackTrack(throwable: Throwable)
    abstract fun stackTrack(throwable: Throwable, force: Boolean)
    abstract fun logUser(userId: String, userName: String)
    abstract fun logUser(userId: Long, userName: String)
    abstract fun longInfo(tag: String, str: String)
}
